paper/tex/methods.tex

```diff
@@ -1,163 +1,164 @@
 \section{Methods}
```

Comment:
You moved the big chi^2 equation from 3.1 to 3.1.1 for LePhare.  The move to make it more general was in response to an earlier referee comment, so I would restore it back to 3.1 and keep the language saying that the chi^2 calculation underpins all 3 template codes.